Packs/Core/ReleaseNotes/3_2_10.md

```diff
@@ -0,0 +1,34 @@
+
+#### Playbooks
+
+##### New: Remote WMI Process Execution
+
+- New: This playbook addresses the following alerts:
+
+* Remote WMI process execution
+* Suspicious remote WMI process execution
+
+**Playbook Stages:**
+
+**Enrichment:**
+
+* Enrich the attacker’s IP address to identify any known malicious activity.
+
+* Retrieve all incident-related alerts to consolidate context for further analysis.
+
+**Investigation:**
+
+* Analyze command-line activity to assess risks based on suspicious patterns.
+
+* Check for high-confidence evidence, such as malicious IP addresses or suspicious command-line activity, to determine the next course of action.
+
+* Evaluate medium-confidence detections and request analyst approval for further containment if required.
+
+**Containment:**
+
+* Attempt to terminate the malicious process tree using its causality ID.
+
+* Provide guidance for manual process termination if the automated action fails.
+
+* Propose endpoint isolation to prevent further compromise if malicious activity is confirmed.<~XSIAM> (Available from Cortex XSIAM %%XSIAM_VERSION%%).</~XSIAM>
```

Comment:
Should 
<~XSIAM> (Available from Cortex XSIAM %%XSIAM_VERSION%%).</~XSIAM> 
be removed?

Reply:
Yes. Fixed.